postgres/README.md

```diff
@@ -119,6 +122,30 @@ PostgreSQL default logging is to `stderr` and logs do not include detailed infor
       #log_destination = 'eventlog'
     ```
 
+2. To gather detailed duration metrics and make them searchable in the Datadog interface, they will need to be 
+   configured inline with the statement themselves.  See below for the recommended config, note that counterintuitively 
+   both `log_statement` and `log_duration` options are commented out.
+   
+   This config will log all statements, but to optionally reduce the output to those which have a certain duration,
+   set the `log_min_duration_statement` value to the desired minimum ms:
+
+    ```conf
+      logging_collector = on
+      log_directory = 'pg_log'  # directory where log files are written,
+                                # can be absolute or relative to PGDATA
+      log_filename = 'pg.log'   #log file name, can include pattern
+      log_min_duration_statement = 0    # -1 is disabled, 0 logs all statements
```

Comment:
Should we keep `-1` as a default, so that users don't inadvertently start sending all their queries to Datadog right after copy-pasting the config?

Reply:
If they turn on logging with the previous stanza, they would be sending all the queries already, this change just adds the duration field.

They will need to restart the server before the changes take effect anyway, so there is some protection from inadvertently uncommenting something.